app/test/consistent_apphash_test.go

```diff
@@ -388,7 +440,7 @@ func executeTxs(testApp *app.App, encodedBlobTx []byte, encodedSdkTxs [][]byte,
 	dataHash := resPrepareProposal.BlockData.Hash
 
 	header := tmproto.Header{
-		Version:        version.Consensus{App: 1},
+		Version:        version.Consensus{App: appVersion},
```

Comment:
```suggestion
		Version:        version.Consensus{App: testApp.AppVersion()},
```
You could just do this which avoids needing the extra argument